nixd/docs/user-guide.md

```diff
@@ -64,6 +64,17 @@ nix build -L .#
 
 ### Configuration
 
+`.nixd.json`, the configuration of `nixd`, supports [json schema](https://json-schema.org/).
```

Comment:
Maybe better write these paragraphs after the description.

Reply:
Done.